README.md

```diff
@@ -1,8 +1,6 @@
-# merlin [![](https://ci.novoda.com/buildStatus/icon?job=merlin)](https://ci.novoda.com/job/merlin/lastBuild/console) [![Download](https://api.bintray.com/packages/novoda/maven/merlin/images/download.svg) ](https://bintray.com/novoda/maven/merlin/_latestVersion) [![](https://raw.githubusercontent.com/novoda/novoda/master/assets/btn_apache_lisence.png)](LICENSE.txt)
+![Merlin header image](https://github.com/novoda/merlin/blob/master/readme-sources/header.png)
 
-An OK name for a library.
-
-## Description
+# Merlin [![](https://ci.novoda.com/buildStatus/icon?job=merlin)](https://ci.novoda.com/job/merlin/lastBuild/console) [![Download](https://api.bintray.com/packages/novoda/maven/merlin/images/download.svg) ](https://bintray.com/novoda/maven/merlin/_latestVersion) [![](https://raw.githubusercontent.com/novoda/novoda/master/assets/btn_apache_lisence.png)](LICENSE.txt)
```

Comment:
`btn_apache_lisence`? lol dat typo, can we fix?
Also there's an extra space in the download button link: `...download.svg) ]`